testing/src/test/java/io/stargate/it/MultipleStargateInstancesTest.java

```diff
@@ -63,10 +66,13 @@ public void setup(TestInfo testInfo) {
             .withString(
                 DefaultDriverOption.LOAD_BALANCING_POLICY_CLASS,
                 DcInferringLoadBalancingPolicy.class.getName())
-            .withDuration(DefaultDriverOption.REQUEST_TIMEOUT, Duration.ofSeconds(30))
-            .withDuration(DefaultDriverOption.CONNECTION_INIT_QUERY_TIMEOUT, Duration.ofSeconds(30))
-            .withDuration(DefaultDriverOption.CONTROL_CONNECTION_TIMEOUT, Duration.ofSeconds(30))
-            .withDuration(DefaultDriverOption.REQUEST_TRACE_INTERVAL, Duration.ofSeconds(1))
+            .withDuration(DefaultDriverOption.REQUEST_TIMEOUT, Duration.ofSeconds(180))
```

Comment:
Does this still need to be here?

Reply:
Not sure, I think I just standardized it across all the test suites